### PR TITLE
Adding nn.MV

### PR DESCRIPTION
**Pull request by ebetica**
Adding matrix vector multiplication in nn, following the conventions of nn.MM.

### CODE REVIEW

File: MV.lua [hunk 1]
```diff
@@ -0,0 +1,78 @@
+--[[ Module to perform matrix vector multiplication on two minibatch inputs,
+producing a minibatch.
+]]
+
+local MV, parent = torch.class('nn.MV', 'nn.Module')
+
+function MV:__init(trans)
+  parent.__init(self)
+
+  self.trans = trans or false
```
Comment (soumith) [inline MV.lua]: assert that this is boolean.

File: MV.lua [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  self.gradInput = {torch.Tensor(), torch.Tensor()}
+end
+
```
Comment (soumith) [inline MV.lua]: local unpack  = unpack or table.unpack, and use that function in the rest of the module. this is for Lua back-compatibility reasons.

**Comment by soumith** [comment]
minor in-line comments, looks good otherwise.

**Comment by ebetica** [comment]
Fixed.

**Comment by soumith** [comment]
Thanks Zeming!

**Pull request merged**